src/GameLogic/ItemPriceCalculator.cs

```diff
@@ -52,6 +66,10 @@ public class ItemPriceCalculator
                 {
                     baseprice = 2000;
                 }
+                else if (item.Level == 3)
+                {
+                    baseprice = 2800;
+                }
```

Comment:
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/common/zzzitem.cpp#L1863
The client recognizes Arrows/Bolts +3, but I don't know if you are even supposed to get them :smile: